client/src/main/scala/com/github/dapperware/slack/api/SlackRemoteFiles.scala

```diff
@@ -0,0 +1,33 @@
+package com.github.dapperware.slack.api
+
+import com.github.dapperware.slack.{ SlackEnv, SlackError }
+import com.github.dapperware.slack.models.RemoteFile
+import zio.ZIO
+import sttp.client3._
+
+trait SlackRemoteFiles {
+  def addRemoteFile(
+    externalId: String,
+    externalUrl: String,
+    title: String,
+    fileType: Option[String] = None,
+    indexableFileContents: Option[Array[Byte]] = None,
```

Comment:
Can we use `Chunk` here instead of a raw array?

Reply:
done